radis/lbl/base.py

```diff
@@ -1526,6 +1539,52 @@ def _calc_noneq_parameters(self, vib_distribution, singleTvibmode):
             self.calc_weighted_trans_moment()
             self.calc_einstein_coefficients()
 
+        # Caching the non-equilibrium parameters Evib and Erot
+        # First, getting the Electronic States file through partition functions:
+        for isotope in self._get_isotope_list(molecule):
+            elec_state = self.get_partition_function_calculator(
+                molecule, isotope, state
+            ).ElecState
+            # fname = "{0}_{1}_{2}_EvibErot".format(elec_state.jsonfile, molecule, isotope)
+            fname = elec_state.jsonfile
+            f1 = splitext(elec_state.jsonfile)[0] + "_{0}_{1}_EvibErot".format(
+                molecule, isotope
+            )
+            fcache = DataFileManager(engine).cache_file(f1)
+            # Generate cache file for later use
+            if cache:
+                new_metadata = {
+                    # Last modification time of the original file :
+                    "last_modification": time.ctime(getmtime(fname)),
+                    "Evibu": df.Evibu,
+                    "Evibl": df.Evibl,
```

Comment:
You shouldn't cache the energies in the metadata. Energies potentially tens of MB and metadata should be fast to load and compare.
Keep only the spectroscopic constant file name, and last modification date